docs/api/io/BufferedSerial.md

```diff
@@ -0,0 +1,71 @@
+# BufferedSerial
+
+<span class="images">![](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_buffered_serial.png)<span>`BufferedSerial` class hierarchy</span></span>
+
+The `BufferedSerial` class provides UART functionality, it is the recommended class for serial data transfers. It allows sending and receiving bytes of data in a sequence using separate transmit (TX) and receive pins (RX). Communication can be between two processors or for sending text to a console.
+
+TX and RX pins can be specified as Not Connected (NC) for simplex (unidirectional) communication or as valid pins for full duplex (bi-directional) communication.
+
+Data is sent/received at a pre-defined speed called baud rate. Standard baud rate include 9600, 119200, 115200 or others.
```

Comment:
```suggestion
`Baud rate` - Data is sent and received at a pre-defined speed called baud rate. Standard baud rates include 9600, 119200, 115200 or others.
```

Reply:
That `Baud rate  -` does not flow with the rest. I am taking the other changes.